commons/src/test/java/com/ozonehis/eip/utils/ShutdownHandlerTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © 2021, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.utils;
+
+import static org.mockito.Mockito.*;
+
+import org.junit.jupiter.api.*;
```

Comment:
Remove the wildcards.
You to set your IDE to not use wildcards. For IntelliJ, `File > Settings > Editor > Code Style > Java > Imports > Class count to use import with '*'`